src/vs/workbench/api/node/mainThreadMessageService.ts

```diff
@@ -58,4 +67,27 @@ export class MainThreadMessageService extends MainThreadMessageServiceShape {
 			});
 		});
 	}
+
+	private showModalMessage(severity: Severity, message: string, commands: { title: string; isCloseAffordance: boolean; handle: number; }[]): Thenable<number> {
+		let closeAffordanceIndex = -1;
```

Comment:
Just keep a `boolean`? I do not see `closeAffordanceIndex` being used for anything but that. Or is that an oversight?